Grid.elm

```diff
@@ -10,22 +11,15 @@ import Matrix.Extra
 -- Battleship
 import Ship
 
-
 -- Grid
 type alias IsHit = Bool
 type alias Grid = Matrix.Matrix Cell
 type Cell
     = Ship IsHit
     | Empty IsHit
+    -- TODO: Sunk state
```

Comment:
Not quite. There should be also an indication that a ship has been fully destroyed (every cell of a ship was hit)